src/applications/gi/components/vet-tec/VetTecContactInformation.jsx

```diff
@@ -44,12 +44,22 @@ export const VetTecContactInformation = ({ institution }) => (
     {/* Production flag for 19534 */}
     {!environment.isProduction() &&
       institution.schoolCertifyingOfficials[0] && (
-        <div className="additional-information row vads-u-margin-y--4">
-          {VetTecScoContact(
-            institution.schoolCertifyingOfficials[0],
-            'School certifying officials',
+        <div>
+          <div className="vads-u-margin-top--4">
+            <h3>School certifying officials</h3>
+          </div>
+          {institution.schoolCertifyingOfficials.map(
+            (sco, i) =>
+              i % 2 === 0 && (
```

Comment:
were we not able to use [flexbox grid](https://design.va.gov/layout/flexbox-grid) to accomplish this? I think mapping over every other item this way is discouraged.

Reply:
Hi Rian, thanks for the suggestion. I agree that the "every other" index mapping solution was not ideal. I just pushed a commit using flexbox grid as you suggested to display items in  `VetTecContactInformation.jsx`